src/components/BuildName/BuildName.tsx

```diff
@@ -53,8 +56,23 @@ interface IConfigAppendix {
 
 const ConfigAppendix = ({ build, includeConfigLink }: IConfigAppendix) => {
   const configName = calculateBuildConfigName(build);
-  const configLink = '/TODO'; // TODO: FORMAT LINKS HERE
-  return <> of {includeConfigLink ? <Link to={configLink}>{configName}</Link> : configName}</>;
+
+  let configLink;
+  if (isBuild(build)) {
+    const recognizedBuild: Build = build;
+    configLink = (
+      <BuildConfigLink id={recognizedBuild.buildConfigRevision?.id!} rev={recognizedBuild.buildConfigRevision?.rev!}>
+        {configName}
+      </BuildConfigLink>
+    );
+  } else if (isGroupBuild(build)) {
+    const recognizedGroupBuild: GroupBuild = build;
+    configLink = <GroupConfigLink id={recognizedGroupBuild.groupConfig?.id!}>{configName}</GroupConfigLink>;
+  } else {
+    uiLogger.error('Build entity was not recognized');
```

Comment:
do we set the config link to EmptyStateSymbol if it is not a recognized build?

Reply:
I would rather avoid it for now. I put there this checker to test it, but we need to come with some more general solution once Build vs BuildRef dicussion is finished